### PR TITLE
OCPQE-20701, OCPCLOUD-2619 - cluster-capi-operator rebase tests

### PR DESCRIPTION
**Pull request by miyadav**
/pj-rehearse pull-ci-openshift-cluster-capi-operator-main-regression-clusterinfra-cucushift-rehearse-capi-aws-ipi

### CODE REVIEW

File: ci-operator/config/openshift/cluster-capi-operator/openshift-cluster-capi-operator-main.yaml [hunk 1]
```diff
@@ -7,6 +7,10 @@ base_images:
     name: builder
     namespace: ocp
     tag: rhel-9-golang-1.22-openshift-4.17
+  tests-private:
+    name: tests-private
+    namespace: ci
+    tag: "4.16"
```
Comment (damdo) [inline ci-operator/config/openshift/cluster-capi-operator/openshift-cluster-capi-operator-main.yaml:13]: What does this tag imply?
Will this be automatically bumped?

Comment (miyadav) [reply ci-operator/config/openshift/cluster-capi-operator/openshift-cluster-capi-operator-main.yaml:13]: I will need to check this , at present we are manually doing it . This is the release version of our test repo.
 @liangxia @kuiwang02 let me know if I am not correct.

Comment (liangxia) [reply ci-operator/config/openshift/cluster-capi-operator/openshift-cluster-capi-operator-main.yaml:13]: Yes, we are changing this manually. 
This indicate which image/branch the ginkgo e2e tests (openshift-extended-test) is using.

**Comment by miyadav** [comment]
@damdo PTAL , when time permits.

File: ci-operator/config/openshift/cluster-capi-operator/openshift-cluster-capi-operator-main.yaml (hunk 1 above)
Comment (damdo) [reply ci-operator/config/openshift/cluster-capi-operator/openshift-cluster-capi-operator-main.yaml:13]: Ok got it. Do you have reminders set up to remind you to do this then?

**Comment by damdo** [comment]
@miyadav  LGTM but you'll need to also update release-4.17 and release-4.18 files and rerun job generation.

File: ci-operator/config/openshift/cluster-capi-operator/openshift-cluster-capi-operator-main.yaml (hunk 1 above)
Comment (miyadav) [reply ci-operator/config/openshift/cluster-capi-operator/openshift-cluster-capi-operator-main.yaml:13]: yes when the branch cuts ( code freeze for openshift ) , we also branch out from our master and then update . So for example , our master at present will become 4.17 , when openshift master cuts to 4.17 , not exactly the same time but close to it .

**Comment by damdo** [comment]
@miyadav I mean updating https://github.com/openshift/release/blob/master/ci-operator/config/openshift/cluster-capi-operator/openshift-cluster-capi-operator-release-4.17.yaml and https://github.com/openshift/release/blob/master/ci-operator/config/openshift/cluster-capi-operator/openshift-cluster-capi-operator-release-4.18.yaml the same way you did with main.yaml

Thanks!

**Comment by miyadav** [comment]
thanks @damdo , done.

**Comment by miyadav** [comment]
/pj-rehearse pull-ci-openshift-cluster-capi-operator-main-regression-clusterinfra-cucushift-rehearse-capi-aws-ipi

**Comment by miyadav** [comment]
/pj-rehearse pull-ci-openshift-cluster-capi-operator-main-regression-clusterinfra-cucushift-rehearse-capi-aws-ipi

**Review by damdo — COMMENTED** [review]
/approve
/lgtm

**Comment by damdo** [comment]
@miyadav will need rebase
and rehearsal ack

**Review by damdo — COMMENTED** [review]
/lgtm

**Comment by miyadav** [comment]
/pj-rehearse pull-ci-openshift-cluster-capi-operator-main-regression-clusterinfra-cucushift-rehearse-capi-aws-ipi

**Pull request merged**